docs/apm-infra-correlation.md

```diff
@@ -0,0 +1,295 @@
+# How to Enable APM-Infra Correlation 
+
+In the APM Service Dashboards, there are charts that indicate the health of
+the underlying infrastructure. The default configuration of Splunk OpenTelemetry 
+Connector automatically configures this for you, but when specifying a custom 
+configuration it is important to keep the following in mind.
+
+
+
+## Configuration to enable APM-Infra Correlation from a standalone VM (Agent)
+In order to get your infrastructure data to show up, you need to enable the 
+following in the otel-collector:
+
+- `hostmetrics` receiver
+    - cpu, memory, filesystem and network enabled
+   
+    <em>This will allow the collection of cpu, mem, disk and network metrics.</em>
+
+
+- `signalfx` exporter 
+    
+    <em>The `signalfx` exporter will aggregate the metrics from the `hostmetrics` 
+    receiver and send in metrics such as `cpu.utilization`, which are referenced in 
+    the relevant APM service charts.</em>
+
+    - `correlation` flag (enabled by default)
+
+    - `signalfx` exporter must be enabled for metrics AND traces pipeline (example below)
+
+    <em>The `correlation` flag of the `signalfx` exporter will allow the 
+    collector to make relevant API calls to correlate your spans with the 
+    infrastructure metrics, but to do this the `signalfx` exporter must be placed 
+    in the traces pipeline. This flag is enabled by default, so no specific configuration
+    is needed for it. You can, however, fine tune the `correlation` option further if 
+    needed. [Here](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/signalfxexporter#traces-configuration-correlation-only) are the relevant docs to tune 
+    the `correlation` option.</em>
+
+- `resourcedetection` processor 
+    - cloud provider/env variable to set `host.name`
+    - override enabled
+    
+    <em>This processor will enable a unique `host.name` value to be set for metrics 
+    and traces. The `host.name` is determined by either the ec2 host name, or the 
+    system host name.</em>
+   
+
+-  `resource/add_environment` processor (optional)
+    
+    <em>This processor inserts a `deployment.environment` span tag to all the spans. The APM
```

Comment:
Suggested changes:

- "... to all spans."
- "... span tag to be set correctly." 
- "You should configure this span tag in the instrumentation, but if that is not an option, you can use this processor ..."

Reply:
ack